scripts/fetchODCDataFiles.js

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2020 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/inclusive-design/data-update-github/main/LICENSE
+*/
+
+// This script checks if a new data file is published on the data source URL. If there is,
+// the script downloads the file, updates the corresponding latest.json and issues a pull request
+// against [the COVID data repository](https://github.com/inclusive-design/covid-assessment-centres/).
+//
+// Prerequisites:
+// Before running this script, an environment variable GITHUB_ACCOUNT_URL must be defined . This variable defines
+// the Github repository URL with an embedded authenticated github account that pull requests will be issued on behalf.
+// An example: https://{username}:{personal-access-token}@github.com/wecountproject/covid-assessment-centres.git
+//
+// A sample command that runs this script in the universal root directory:
+// node scripts/fetchODCDataFiles.js https://data.ontario.ca/dataset/covid-19-assessment-centre-locations ODC
+
+"use strict";
+
+const fs = require("fs");
+const rimraf = require("rimraf");
+const utils = require("./fetchODCDataFilesUtils.js");
+
+const wecountprojectRepoUrl = process.env.GITHUB_ACCOUNT_URL;
+
+if (!wecountprojectRepoUrl) {
+	console.log("Error: Please define an environment variable \"GITHUB_ACCOUNT_URL\" with a value of an authenticated Github account URL that will be used to create remote branches on behalf of this account.\n");
+	console.log("The command to define an temporary environment variable in a terminal: export GITHUB_ACCOUNT_URL={value}\n");
+	console.log("An example of the environment variable value: https://{username}:{personal-access-token}@github.com/wecountproject/covid-assessment-centres.git");
+	process.exit(1);
+}
+
+var dataSourceURL = process.argv[2];
+var dataDirInRepo = process.argv[3];
+
+if (process.argv.length < 3) {
+	console.log("Usage: node scripts/fetchODCDataFiles.js dataSourceURL dataDirInRepo\n");
+	console.log("dataSourceURL: The URL to webpage where the data file for COVID-19 assessment centre locations is published. An example: https://data.ontario.ca/dataset/covid-19-assessment-centre-locations");
+	console.log("dataDirInRepo: The directory where the downloaded data file should be placed in the COVID-19 data repository(https://github.com/inclusive-design/covid-assessment-centres/). This directory is relative to the root directory of the repository ");
+	process.exit(1);
+}
+
+// The central COVID-19 data repository on Github
+const covidDataRepoUrl = "https://github.com/cindyli/covid-assessment-centres.git";
```

Comment:
Should this URL be hardcoded? I remember when we first spoke about this feature we agreed to endpoints like this should be lifted into a JSON configuration file. Same might be appropriate for the dataSourceURL and dataDirInRepo above.